### PR TITLE
Add Glu styling

### PR DESCRIPTION
**Pull request by samjowen**
Closes https://github.com/Capventis/gluplatform-login-ui/issues/1.

This is a transfer of the Glu styling from the old repo: https://github.com/Capventis/gluplatform-login.

https://github.com/Capventis/gluplatform-app/issues/1161

---

Hi Sim, 

This code should all be working in Docker. Please let me know if you can spot anything that I have overlooked. 👍🏻 

Thanks,
Sam

### CODE REVIEW

File: pages/error.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Link from "next/link"
 import { useRouter } from "next/router"
 import { useEffect, useState } from "react"
 
-import { ActionCard, CenterLink, MarginCard } from "../pkg"
+import { GluActionCard, GluCenterLink, GluMarginCard } from "../pkg"
```
Comment (simeon-walker) [inline pages/error.tsx:9]: Is there a reason why you added the GLu prefix to lots of these things?

File: pages/verification.tsx [hunk 2]
```diff
@@ -66,58 +72,50 @@ const Verification: NextPage = () => {
...
 
-  const onSubmit = async (values: SubmitSelfServiceVerificationFlowBody) => {
-    await router
+  const onSubmit = (values: SubmitSelfServiceVerificationFlowBody) =>
+    router
       // On submission, add the flow ID to the URL but do not navigate. This prevents the user loosing
-      // their data when they reload the page.
+      // his data when she/he reloads the page.
       .push(`/verification?flow=${flow?.id}`, undefined, { shallow: true })
+      .then(() =>
```
Comment (simeon-walker) [inline pages/verification.tsx:80]: Can you explain what's going on in the changes around here?

File: pages/error.tsx (hunk 1 above)
Comment (samjowen) [reply pages/error.tsx:9]: This was just to indicate that the Glu-prefixed components were [styled-components](https://styled-components.com/) that I had overridden in the pkg/styled/index.tsx file.

I was thinking that we could eventually have a library of styled components that all had different prefixes depending on the project.

File: pages/verification.tsx (hunk 2 above)
Comment (samjowen) [reply pages/verification.tsx:80]: I think I have pasted in an older file - it looks like the Ory people have made some changes in the time that I cloned the repo and when you forked it. The inclusive language stuff is definitely not my writing and the he/she stuff seems to be the older version as 'they' is the done way to do things nowadays.

Good catch, I think we should probably keep the newer file.

Comment (simeon-walker) [reply pages/verification.tsx:80]: We could do a new branch and then git merge their master, just to see what they did.

Comment (simeon-walker) [reply pages/verification.tsx:80]: Not seeing any changes, but either way just make the file so the diff is just the changes you intended to make.

**Review by simeon-walker — APPROVED** [review]
Resulting diff looks better now.

**Pull request merged**